shard-operator/src/test/java/com/redhat/service/smartevents/shard/operator/ManagerSyncServiceTest.java

```diff
@@ -231,4 +244,45 @@ public void testProcessorsAreDeletedWhenNotDeployed() throws JsonProcessingExcep
         assertJsonRequest(expectedJsonUpdateRequestForDeprovisioning, APIConstants.SHARD_API_BASE_PATH + "processors");
         assertJsonRequest(expectedJsonUpdateRequest, APIConstants.SHARD_API_BASE_PATH + "processors");
     }
+
+    @Test
+    public void bridgeMetricsAreProduced() throws JsonProcessingException, InterruptedException {
+        doBridgeDeployment();
+
+        String metrics = given()
+                .filter(new ResponseLoggingFilter())
+                .contentType(ContentType.JSON)
+                .when()
+                .get("/q/metrics")
+                .then()
+                .extract()
+                .body()
+                .asString();
+
+        // Not all metrics are recorded by this test
+        assertThat(metrics).contains(managerRequestsTotalCountMetricName);
```

Comment:
What is the reason for not covering all the metrics?

Reply:
The test only handles successful creation; but one metric relates to failures.

Reply:
Would it be also possible to add some test which simulates failure?

Reply:
Done.